### PR TITLE
 Make Helm sync GH workflow add Helm Chart docs, too

### PR DESCRIPTION
**Pull request by Annopaolo**
The Helm Chart did not include a README.md file. Add Helm Chart docs generation from sources using [helm-docs](https://github.com/norwoodj/helm-docs) in the Helm sync workflow.

### CODE REVIEW

**Comment by Annopaolo** [comment]
Blocked until #260 is merged.

**Review by matt-mazzucato — APPROVED** [review]

**Review by bettio — APPROVED** [review]

**Pull request merged**
